repository-ydb-v2/src/test/java/tech/ydb/yoj/repository/ydb/YdbRepositoryIntegrationTest.java

```diff
@@ -1355,6 +1357,42 @@ public void schemaNotExistsForMissingEntity() {
         assertThat(repository.schema(MissingEntity.class).exists()).isFalse();
     }
 
+    @Test
```

Comment:
These tests should be pulled up to RepositoryTest - In-memory DB works the same, so they should still pass.

Reply:
done